workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -170,30 +170,30 @@ def detect_changes(self, branch, event, save_tags=False):
             )
 
             changed = False
-            # look for changed tags
-            for tag, reason in tags_to_check:
-                if tags_new[tag] != tags_old[tag]:
+            # look for changed metadata
+            for key, reason in metadata_to_check:
+                if new_metadata[key] != old_metadata[key]:
                     changes.append(reason)
                     changed = True
 
             if changed:
-                if save_tags:
-                    # we may not want to update the tags
-                    event.repository_tags = self.serialize(tags_new)
+                if save_metadata:
+                    # we may not want to update the metadata
+                    event.repository_metadata = self.serialize(new_metadata)
 
-                event.tag_changes_detected = "\n".join(changes)
-                event.tags_changed = True
+                event.metadata_changes_detected = "\n".join(changes)
```

Comment:
This is probably just grammar, but I would expect a variable called `metadata_changes_detected` to be a Boolean ("have metadata changes been detected, yes or no?") rather than a long string.  Maybe `metadata_all_changes` ?